tests/testthat/test-utils.R

```diff
@@ -196,75 +220,40 @@ test_that("makePanels() does nothing if there are no facets", {
 })
 
 test_that("contingencyDT() returns appropriately sized data.table", {
-  dt <- contingencyDT(data.binned)
+  # dt <- contingencyDT(data.binned)
+  dt <- contingencyDT(test.df[, c('entity.cat3','entity.cat4','entity.cat7','entity.cat6')])
 
   expect_is(dt, 'data.table')
-  expect_equal(nrow(dt),data.table::uniqueN(data.binned$x))
-  expect_equal(length(dt),data.table::uniqueN(data.binned$y)+1)
+  expect_equal(nrow(dt),data.table::uniqueN(test.df$cat7))
+  expect_equal(length(dt),data.table::uniqueN(test.df$cat6)+1)
 })
 
 
-test_that("remapListVar appropriately updates map", {
```

Comment:
same question. did this move?

Reply:
The remapListVar function was used in one of the earlier ways we handled listvars, but is no longer used. I should also delete it from `utils.R`.